main.py

```diff
@@ -57,63 +66,68 @@ def check(m):
 
     await ctx.send("Select an amount of time for the giveaway.")
     try:
-        since = await bot.wait_for('message', check=check, timeout=30.0)
+        since = await bot.wait_for('message', check = check, timeout=30.0)
 
     except asyncio.TimeoutError:
         await ctx.send("You took to long, please try again!")
+        
+    
+
 
     seconds = ("s", "sec", "secs", 'second', "seconds")
-    minutes = ("m", "min", "mins", "minute", "minutes")
-    hours = ("h", "hour", "hours")
+    minutes= ("m", "min", "mins", "minute", "minutes")
+    hours= ("h", "hour", "hours")
     days = ("d", "day", "days")
-    weeks = ("w", "week", "weeks")
     rawsince = since.content
-
+    
+    
     try:
         temp = re.compile("([0-9]+)([a-zA-Z]+)")
         if not temp.match(since.content):
             return await ctx.send("You did not specify a unit of time, please try again.")
+
         res = temp.match(since.content).groups()
         time = int(res[0])
         since = res[1]
-
+        
     except ValueError:
         return await ctx.send("You did not specify a unit of time, please try again.")
-
+        
     if since.lower() in seconds:
         timewait = time
     elif since.lower() in minutes:
-        timewait = time * 60
+        timewait = time*60
     elif since.lower() in hours:
-        timewait = time * 3600
+        timewait = time*3600
     elif since.lower() in days:
-        timewait = time * 86400
+        timewait = time*86400
     elif since.lower() in weeks:
-        timewait = time * 604800
+        timewait = time*604800
     else:
-
+        
         return await ctx.send("You did not specify a unit of time, please try again.")
-
+        
     await ctx.send("What would you like the prize to be?")
     try:
-        msg4 = await bot.wait_for('message', check=check, timeout=30.0)
+        msg4 = await bot.wait_for('message', check = check, timeout=30.0)
+
 
     except asyncio.TimeoutError:
         await ctx.send("You took to long, please try again.")
 
+
     logembed = discord.Embed(title="Giveaway Logged",
                              description=f"**Prize:** ``{msg4.content}``\n**Winners:** ``{winerscount}``\n**Channel:** {giveawaychannel.mention}\n**Host:** {ctx.author.mention}",
                              color=discord.Color.red())
     logembed.set_thumbnail(url=ctx.author.avatar_url)
 
-    logchannel = ctx.guild.get_channel(609431364445405194)  # Put your channel, you would like to send giveaway logs to.
+    guild = client.get_guild(796081913688883240)  # Put your guild ID here!

```

Comment:
client?

Reply:
Sorry, I usually use client, it's more updated from the word bot.